### PR TITLE
docs(colors): get color from element

### PR DESCRIPTION
**Pull request by Fridaks**
## 📥 Proposed changes

affects: @fremtind/portal

changed from hardcoded color to picking out the color using getComputedProperty

ISSUES CLOSED: #1399

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by lfbergee** [comment]
@all-contributors please add @Fridaks for code

File: portal/src/components/Documentation/Typography/TypograhyTable/ExampleRow.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { TableCell } from "./TableCell";
+import { getComputedProperty } from "../../../../utils/getComputedProperty";
```
Comment (piofinn) [inline portal/src/components/Documentation/Typography/TypograhyTable/ExampleRow.tsx:3]: Skal vi vurdere å legge til [`gatsby-plugin-root-import`](https://www.gatsbyjs.com/plugins/gatsby-plugin-root-import/)? Begynner å se en del sånne lange relative stier i portalen.

**Review by piofinn — APPROVED** [review]
Nice! Digg å få fikset dette for fargene også.

Velkommen som bidragsyter, @Fridaks! 🎉

File: portal/src/components/Documentation/Typography/TypograhyTable/ExampleRow.tsx (hunk 1 above)
Comment (lfbergee) [reply portal/src/components/Documentation/Typography/TypograhyTable/ExampleRow.tsx:3]: Kanskje, men ikke her hvertfall :p

**Review by lfbergee — APPROVED** [review]

**Pull request merged**